### PR TITLE
Add DSpace timeout to config and refactor sample data loading process

### PR DESCRIPTION
**Pull request by hakbailey**
#### Includes new or updated dependencies?
YES

#### Changes expectations for external applications?
NO

#### Developer
- [x] All new ENV is documented in README
- [x] All new ENV has been added to staging and production environments
- [x] All related Jira tickets are linked in commit message(s)

#### How can a reviewer manually see the effects of these changes?
Using the instructions in the README for developing locally with moto_server, run a local sqs instance, create input and output queues, and then load sample data using the "integration-test-submission-messages" fixture. Then while connected to the VPN and with the appropriate DSpace credentials set in .env, run `pipenv submitter start`. You will see that one thesis is successfully posted to DSpace test.

Now, set DSPACE_TIMEOUT=1.0 in your .env and run the entire process again. You will see that the error messages have changed to a timeout error, which is the desired result from this PR. 

Note: You will also notice that several of the theses from the second run have now posted to DSpace, because DSS can't delete posted items from DSpace if DSpace doesn't return a handle. This is the expected behavior for now, and there is a separate ticket to ensure that (with a long timeout set, as we will have in production) any response failure from the DSpace server crashes DSS and stops the processing of more messages, so we don't end up with a bunch of partially-posted items in DSpace.

#### Code Reviewer
- [ ] The commit message is clear and follows our guidelines
      (not just this pull request message)
- [ ] There are appropriate tests covering any new functionality
- [ ] The documentation has been updated or is unnecessary
- [ ] The changes have been verified
- [ ] New dependencies are appropriate or there were no changes

### CODE REVIEW

File: submitter/sqs.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def message_loop(queue, wait, visibility=30):
 
 def process(msgs):
     if config.SKIP_PROCESSING != "true":
-        client = DSpaceClient(config.DSPACE_API_URL)
+        client = DSpaceClient(config.DSPACE_API_URL, timeout=config.DSPACE_TIMEOUT)
```
Comment (osmandin) [inline submitter/sqs.py:36]: @hakbailey The change looks good to me, but is there an exception raised from Dspace (client) and caught (for logging, e.g.,) if there's a login timeout?

Comment (JPrevost) [reply submitter/sqs.py:36]: The DSpace client library does throw an appropriate exception in this case:
https://github.com/MITLibraries/dspace-python-client/pull/14/files

Making the exceptions more visible by implementing Sentry is part of a separate ticket in this current Sprint:
https://mitlibraries.atlassian.net/browse/ETD-443

File: tests/test_cli.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from submitter.cli import main
 
 
-def test_cli_sample_data_loader(mocked_sqs):
+def test_cli_load_sample_data(caplog, mocked_sqs):
```
Comment (JPrevost) [inline tests/test_cli.py:6]: Is caplog automatically passed in via the testing framework?

**Review by JPrevost — APPROVED** [review]

File: tests/test_cli.py (hunk 2 above)
Comment (hakbailey) [reply tests/test_cli.py:6]: Oh, yes but we don't need it now that we're not getting errors around logging in the cli. Will update.

File: submitter/sqs.py (hunk 1 above)
Comment (hakbailey) [reply submitter/sqs.py:36]: Yep, what Jeremy said.

**Pull request merged**